src/main/java/com/postsquad/scoup/web/auth/controller/OAuthController.java

```diff
@@ -16,12 +17,12 @@
     private final OAuthService oAuthService;
 
     @GetMapping("/user-data")
-    public SocialAuthenticationResponse readOAuthUserData(@RequestParam String type, @RequestParam String code) {
+    public SocialAuthenticationResponse readOAuthUserData(@RequestParam(value = "type") OAuthType type, @RequestParam String code) {
```

Comment:
```suggestion
    public SocialAuthenticationResponse readOAuthUserData(@RequestParam OAuthType type, @RequestParam String code) {
```
이렇게만 해도 동작할 것 같은데 잘 안 되나요?

그리고 저도 찾아보다가 https://medium.com/@lifecluee/string%EC%9D%84-%EA%B3%A0%EC%A7%91%ED%95%98%EB%8A%94-controller-%EA%B0%9C%EC%84%A0%EA%B8%B0-721cba570756 이런글을 봤는데 괜찮아보였어요 여유로우실때 한 번 보셔도 좋을 것 같아요 ㅋㅋ